src/map/attackround.cpp

```diff
@@ -358,12 +360,31 @@ void CAttackRound::CreateKickAttacks()
             m_kickAttackOccured = true;
         }
 
-        // TODO: Possible Lua function for the nitty gritty stuff below.
-
-        // Mantra set mod: Try an extra left kick attack.
+        // Tantra set mod: Try an extra left kick attack.
         if (m_kickAttackOccured && dsprand::GetRandomNumber(100) < m_attacker->getMod(Mod::EXTRA_KICK_ATTACK))
         {
             AddAttackSwing(PHYSICAL_ATTACK_TYPE::KICK, LEFTATTACK, 1);
         }
     }
-}
\ No newline at end of file
+}
+
+/************************************************************************
+*																		*
+*  Creates a Daken throw.												*
+*																		*
+************************************************************************/
+void CAttackRound::CreateDakenAttack()
+{
+    if (m_attacker->objtype == TYPE_PC)
+    {
+        // if ammo slot is a shuriken (throwing, consumable)
+        CItemWeapon* PAmmo = m_attacker->m_Weapons[SLOT_AMMO];
+        if (PAmmo && PAmmo->isThrowing()) {
```

Comment:
Brace on new line. But then you did it correctly down below?

Reply:
Because Visual Studio is putting them that way. I was purposefully putting them on new lines. I'm sure it's a setting somewhere.